core/src/css/component/decktracker/overlay/decktracker-cards-recap.component.scss

```diff
@@ -10,7 +10,6 @@
 	width: 60px;
 	height: 30px;
 	border: 1px solid var(--color-1);
-	background: var(--color-5);
```

Comment:
I would like to keep the background

Reply:
I don't think so - and I trust my designer on that.
If I start receiving feedback about it I might consider it, but for now I don't think it needs to change.